src/stories/Library/event-list-item/EventListItem.tsx

```diff
@@ -0,0 +1,49 @@
+// EventListItem.tsx
+import { ReactComponent as ArrowSmallRight } from "../Arrows/icon-arrow-ui/icon-arrow-ui-small-right.svg";
+import { Tag } from "../tag/Tag";
+
+export type EventListItemProps = {
+  image: string;
+  title: string;
+  description: string;
+  date: string;
+  time: string;
+  location: string;
+  price: string;
+  href: string;
+};
+
+export const EventListItem: React.FC<EventListItemProps> = ({
+  image,
+  title,
+  description,
+  date,
+  time,
+  location,
+  price,
+  href,
+}) => {
+  return (
+    <a href={href} className="event-list-item arrow__hover--right-small">
+      <div className="event-list-item__image-container">
+        <img src={image} alt={title} className="event-list-item__image" />
+      </div>
+      <div className="event-list-item__content">
```

Comment:
I wonder if it is possible to drop the `event-list-item__image-container` wrapper.

Perhaps it is just my personal dislike of unnecessary markup :smile:.

Reply:
Hmm i am not sure. 

I want to keep the wrapper, so i can put the css for the background on that element, and if there is no image not need to render any additional element instead of it. 

I am genereally unsure what is the best approach to handle image positioning, also considering we are gonna be using focal point etc. 

I felt this approach, with the css provided to both image-container and the __image itself, would create a layout that would position the image correctly, give the "grid"-structure needed, and have a fallback to the image placeholder with colors incase the image was not there, or something happened in while loading.